src/components/application_manager/test/resumption/resumption_data_test.cc

```diff
@@ -442,7 +488,17 @@ void ResumptionDataTest::SetMenuTitleAndIcon() {
 
   sm::SmartObject sm_title;
   sm_title = "test title";
+
+  if (menu_title_) {
+    delete menu_title_;
+  }
+
   menu_title_ = new sm::SmartObject(sm_title);
+
+  if (menu_icon_) {
+    delete menu_icon_;
```

Comment:
@SNiukalov Smart pointers for all such items?

Reply:
@AByzhynar 
No, because this field we using int the ReturnPointee